cheatle/__main__.py

```diff
@@ -1,2 +1,24 @@
+from curses.ascii import isalpha
+from email.policy import default
```

Comment:
what is this?

Reply:
Good question, I deleted any unnecessary imports